src/app/dashboard/design-edit/design-edit.component.ts

```diff
@@ -13,25 +13,25 @@ export class DesignEditComponent implements OnInit {
 
   ngOnInit() {}
 
-  @ViewChild('editor')
+  @ViewChild('editor', { static: true })
   private emailEditor: EmailEditorComponent;
 
-  editorLoaded(event) {
-    console.log('editorLoaded');
+  editorLoaded(event: any) {
```

Comment:
- in react-email-editor we pass the unlayer object to the callback result; we can make the same here for consistency 
- we can use unlayer-types package for the typings

Reply:
@brunolemos good callouts. Added types + now passing the unlayer object for consistency. Also updated to the modern theme + added the same macbook pro design we have in react-email-editor